aries-site/src/examples/templates/filtering/FilteringCards/FilteringCards.js

```diff
@@ -63,73 +62,83 @@ export const FilteringCards = ({ containerRef }) => {
       <Heading level={2} margin="none">
         Users
       </Heading>
-      <FiltersProvider>
-        <Box gap="medium">
-          <FilterControls
-            data={users}
-            filters={filtersConfig}
-            layerProps={layerProps}
-            searchFilter={{ placeholder: 'Search' }}
-          />
+      <Box fill>
+        <Data
+          data={users}
+          updateOn="change"
+          height={{ min: 'medium', max: '100%' }}
+        >
+          <Toolbar>
+            <DataSearch />
+            <DataFilters layer>
+              <DataFilter property="role" label="Role" />
+              <DataFilter property="status" label="Status" />
+              <DataFilter property="location" label="Location" />
+              <DataFilter property="hoursAvailable" label="Hours Available" />
+              <DataFilter property="name" label="Name" />
+            </DataFilters>
+          </Toolbar>
+          <DataSummary />
           <Users />
-        </Box>
-      </FiltersProvider>
+        </Data>
+      </Box>
     </Box>
   );
 };
 
 const Users = () => {
   const size = useContext(ResponsiveContext);
-  const { filteredResults } = useFilters();
 
   return (
-    <Box overflow="auto" fill>
+    <Box flex overflow="auto">
       <Grid
         columns={
           !['xsmall', 'small'].includes(size)
-            ? 'small'
+            ? [['small', 'xlarge']]
```

Comment:
Why do we set this to "xlarge"?

Reply:
Resolved over 1:1 sync-up. Change ternary operator to `columns={!['xsmall', 'small'].includes(size) ? 'small' : ['auto']}`